src/routes/register/index.js

```diff
@@ -1,88 +1,184 @@
 import { h, Component } from 'preact';
+import Button from '../../components/Button';
+import Header from '../../components/Header';
+import Footer from '../../components/Footer';
+import { Form, Validations } from '../../components/Form';
+import { createClassName } from '../../components/helpers';
+import styles from './styles';
 
-import { asyncForEach, asyncEvery } from '../../components/helpers';
-import style from './style';
-import Header, {
-	Title, Content,
-} from 'components/Header';
-import { Form, InputField, Item } from 'components/input';
-
-import Button from 'components/Button';
-import * as Footer from 'components/Footer';
 
 export default class Home extends Component {
```

Comment:
I think the class name here should be `Register` instead of `Home`.
There is another case like this in the `leaveamessage` route component. Can you please take a look at this too?